meta_test.go

```diff
@@ -268,7 +268,7 @@ func (s *MetaSuite) TestSeries(c *gc.C) {
 func (s *MetaSuite) TestInvalidSeries(c *gc.C) {
```

Comment:
We should have at least one test that tests we can unmarshal a charm with multiple series.